### DISCUSSION TITLE
Cryptocurrency payment from app

### DISCUSSION THREAD

**Discussion by mohammadRezaeian** [mullvad/mullvadvpn-app#9057] [Feature requests & Ideas]
## Issue report

Operating system: Android,Gnu\Linux

App version: 2023.6

### Issue description
Hi
Is it possible please add an option to display payment methods? 
When I want to make a payment, I need to log in to the website and create an address for the payment

**Comment by faern** [comment]
Any specific payment method/option you would like to see?
We are working on adding Google Play Billing payments in the Android app. Other than that and vouchers, the app does not plan to have any payments built in. It's by design that you pay for the service on the website, and not in the app.

**Comment by mohammadRezaeian** [comment]
my wallet is on my phone, and I think it's much simpler. When I want to make a payment, I can create a cryptocurrency payment address from the app instead of logging into the website and doing other tasks.

**Comment by faern** [comment]
We currently have no plans to include cryptocurrency payments integrated in the app. However, we plan to add back the button in the app that can easily take you to the website in a logged in state. That way it becomes just a few clicks to reach the website payments from the app.

**Comment by Rawa** [comment]
Some clarification on this, In Google Play we are not allowed to offer alternative billing methods than Google Billing. It would still be possible in the oss version.
